sale_saleor/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,91 @@
+Saleor Connector for Odoo
+==========================
+
+The ``sale_saleor`` module provides a two-way connector between Odoo and the
```

Comment:
can you make explicit the flows and their direction
- what comes from Saleor
- what comes from Odoo